internal/objects/validator.go

```diff
@@ -347,7 +376,10 @@ func (v *Validator) hasConflict(inst *unstructured.Unstructured) (bool, []string
 			// If the user have chosen not to propagate the object to this descendant,
 			// there shouldn't be any conflict reported here
 			nsLabels := v.Forest.Get(inst.GetNamespace()).GetLabels()
-			if ok, _ := selectors.ShouldPropagate(inst, nsLabels); ok {
+			fmt.Println("fuck1")
```

Comment:
Nope 🤣

Reply:
Ooops I definitely thought I cleaned that up hahah